src/patternfly/components/Tabs/tabs.scss

```diff
@@ -145,6 +145,14 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
       margin-left: 0;
       transform: translateX(0);
     }
+
+    .pf-c-tabs__list {
```

Comment:
I don't think this only needs to apply to `.pf-m-scrollable`. While the react component will add this class to any tabs list that has overflow content, so it's fine to style it this way... for a core user who doesn't have the scroll buttons enabled, and just uses the browser scrolling to scroll a tabs list, they should probably still get the snapping behavior.

Also you could change the axis to "y" in `.pf-m-vertical` tabs.

I'm also wondering if we should create vars for these values so a user can customize/disable them.

Something like

```
--pf-c-tabs__list--ScrollSnapTypeAxis: x;
--pf-c-tabs__list--ScrollSnapTypeBehavior: proximity;
--pf-c-tabs__list--ScrollSnapType: var(--pf-c-tabs__list--ScrollSnapTypeAxis) var(--pf-c-tabs__list--ScrollSnapTypeBehavior);
```

wdyt @mattnolting ?

Reply:
Thank you, where do you think is the best place to put the styles? Do you think I should create some class like `.pf-m-horizontal` here or we just simply apply this to `.pf-c-tabs__list` and `.pf-c-tabs__item` and override them when it comes to `.pf-m-vertical`?

Reply:
The default styles (not having `.pf-m-vertical`) are the horizontal styles, so you can just add them to `__list` and `__item`, then modify them under `.pf-m-vertical`